lib/mix/tasks/merge_projects.ex

```diff
@@ -113,6 +123,36 @@ defmodule Mix.Tasks.Lightning.MergeProjects do
       """)
   end
 
+  defp parse_uuid_mappings(opts) do
+    opts
+    |> Keyword.get_values(:uuid)
+    |> Enum.reduce(%{}, fn mapping_str, acc ->
+      case String.split(mapping_str, ":") do
+        [source_id, target_id] ->
+          source_id = String.trim(source_id)
+          target_id = String.trim(target_id)
+
+          # Support both string and integer keys by adding both forms
```

Comment:
@josephjclark by the way why do we need to add both keys in here? And why are we only parsing the `source` and not the `target`?

Reply:
Hmm, it's actually this part that's failing in the CI. I'll have a look tomorrow

Reply:
This is probably AI silliness that has not been properly validated by this silly human

I do need string and integer types, although maybe there's a better way I can handle this (like maybe my test code should be casting numbers to strings)